.travis.yml

```diff
@@ -1,21 +1,25 @@
 # See https://github.com/silverstripe-labs/silverstripe-travis-support for setup details
+language: php
 
-language: php 
 php:
-  - 5.4
+    - 5.3
+    - 5.4
+    - 7.0
 
 sudo: false
 
-addons:
-  apt:
-    packages:
-    - poppler-utils
-
 env:
   - DB=MYSQL CORE_RELEASE=3.1
-  - DB=MYSQL CORE_RELEASE=3
+
+matrix:
+  include:
+    - php: 5.5
+      env: DB=PGSQL CORE_RELEASE=3
```

Comment:
Probably want to test `PGSQL` with `3`, `3.1`, and `3.2`; and `MYSQL` with `3`, and `3.1` here.